### ISSUE TITLE
Dolfyn 1.3.0 doesn't support scipy 1.14.0

### ISSUE THREAD

**Issue by asaunier-ifr** [lkilcher/dolfyn#129]
Last version of scipy is incompatible with dolfyn 1.3.0.
It is due to the use of  scipy.integrate.cumtrapz which is not anymore present.

Is it planned to deliver a new version of Dolfyn in Pypi ?

Best regards

**Comment by jmcvey3** [comment]
Hi @asaunier-ifr, future development of dolfyn is continuing in [MHKiT](https://github.com/MHKiT-Software/MHKiT-Python). You can install that package with `pip install mhkit` and import using `from mhkit import dolfyn`. Some of the analysis functions have changed, but these should be recorded in the documentation or example notebooks in that package

**Comment by asaunier-ifr** [comment]
Hi @jmcvey3, thanks a lot for this info.
I can see that mhkit is blocking scipy version 1.14.0 for now, but I suppose that limitation will be released as soon as all dependancies will be tested and compatible.
Br

**Comment by jmcvey3** [comment]
Yes, PR https://github.com/MHKiT-Software/MHKiT-Python/pull/333 will fix that.
